### PR TITLE
DOC-2821 ECS Task Definition ARN doc

### PR DESCRIPTION
**Pull request by michaelcretzman**
https://harness.atlassian.net/browse/DOC-2821

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ x] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* https://harness.atlassian.net/browse/DOC-2821

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by thisrohangupta — APPROVED** [review]
Looks good

File: docs/continuous-delivery/onboard-cd/cd-quickstarts/ecs-deployment-tutorial.md [hunk 1]
```diff
@@ -187,20 +187,32 @@ For this tutorial, we'll set up the Task Definition, Service Definition, and Art
...
 
+There are two ways to add the ECS task definition to the Harness service:
+
+- **Task Definition**: Add a connection to the task definition file in a remote Git repo, local [Harness File Store](https://developer.harness.io/docs/continuous-delivery/cd-services/cd-services-general/add-inline-manifests-using-file-store/), or object storage (AWS S3).
+- **Task Definition ARN**: Add the task definition ARN. 
+  - The task definition ARN points to an existing task created and available in the AWS cluster with the required definition.
+  - The task definition will be fetched using the task ARN provided and added to the ECS service configuration provided in the Harness ECS service **Service Definition**.
+  - During deployment, the required task is deployed with the desired count provided in the **Service Definition**. 
+
 If you are new to ECS, please review the AWS documentation on [ECS Task Definitions](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html).
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/onboard-cd/cd-quickstarts/ecs-deployment-tutorial.md:198]:
```suggestion
If you are new to ECS, review the AWS documentation on [ECS Task Definitions](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html).
```

File: docs/continuous-delivery/onboard-cd/cd-quickstarts/ecs-deployment-tutorial.md [hunk 2]
```diff
@@ -187,20 +187,32 @@ For this tutorial, we'll set up the Task Definition, Service Definition, and Art
...
 
+Let's look at an example using a task definition file in the [Harness File Store](https://developer.harness.io/docs/continuous-delivery/cd-services/cd-services-general/add-inline-manifests-using-file-store/).
+
 1. In **Task Definition**, click **Add Task Definition**.  
   You specify what Task Definition to use in the **ECS Task Definition Store**.
 
   ![](./static/ecs-deployment-tutorial-40.png)
 
   You can use a remote repo, but for this tutorial we'll use the built-in Harness file manager, [Harness File Store](../../cd-services/cd-services-general/add-inline-manifests-using-file-store.md).
 2. Click **Harness**, and then click **Continue**.
```
[the remark this answers is not in the document]
Comment (roshnisarangadharan) [reply docs/continuous-delivery/onboard-cd/cd-quickstarts/ecs-deployment-tutorial.md:208]: Adding select for accessibility compatibility

File: docs/continuous-delivery/onboard-cd/cd-quickstarts/ecs-deployment-tutorial.md [hunk 3]
```diff
@@ -187,20 +187,32 @@ For this tutorial, we'll set up the Task Definition, Service Definition, and Art
...
-3. In **Manifest Details**, in **Manifest Name**, enter **Task Definition**.
-4. In **File/Folder Path**, click **Select**. The Harness File Store appears.
-5. Create a new folder named **ECS Tutorial**.
-6. In the new folder, create a new file named **RegisterTaskDefinitionRequest.yaml**.
-7. Paste the following Task Definition into the file, click **Save**, and then click **Apply Selected**.
+3. In **Task Definition**, select **Add Task Definition**.
+4. In **Specify ECS Task Definition Store**, select **Harness**, and select **Continue**.
+5. In **Manifest Details**, enter a name for the task definition.
+6. In **File/Folder Path**, click **Select**. The Harness File Store appears.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/onboard-cd/cd-quickstarts/ecs-deployment-tutorial.md:212]:
```suggestion
6. In **File/Folder Path**, select **Select**. The Harness File Store appears.
```

File: docs/continuous-delivery/onboard-cd/cd-quickstarts/ecs-deployment-tutorial.md [hunk 4]
```diff
@@ -187,20 +187,32 @@ For this tutorial, we'll set up the Task Definition, Service Definition, and Art
...
+7. Create a new folder named **ECS Tutorial**.
+8. In the new folder, create a new file named **RegisterTaskDefinitionRequest.yaml**.
+9. Paste the following Task Definition into the file, click **Save**, and then click **Apply Selected**.
```
Comment (roshnisarangadharan) [inline docs/continuous-delivery/onboard-cd/cd-quickstarts/ecs-deployment-tutorial.md:215]:
```suggestion
9. Paste the following Task Definition into the file, select **Save**, and then select **Apply Selected**.
```

**Review by roshnisarangadharan — APPROVED** [review]
Approved with minor suggestions

**Pull request merged**